Global-API/health.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import FastAPI
+from sqlalchemy import create_engine, exc
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+app = FastAPI()
+
+DATABASE_URL = "TBD"
+
+# Create a SQLAlchemy engine for database connection
+engine = create_engine(DATABASE_URL)
+
+# Create a session maker for SQLAlchemy
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+# Define a base class for SQLAlchemy models
+Base = declarative_base()
+
+@app.get("/health")
+def health_check():
+    try:
+        engine.connnect()
+        engine.close()
+        return {'status': 'ok'}
+    except:
+        return {'status': 'fail'}
```

Comment:
In this case it would be good to return an HTTP error code >= 400, so k8s automatically knows that the service is offline. Best would probably be 503 (service unavailable).

You can still return the JSON object in addition to that.